dev/io.openliberty.xmlws.3.0.internal.tools/bnd.bnd

```diff
@@ -0,0 +1,65 @@
+*#*******************************************************************************
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-Name: io.openliberty.xmlWS.3.0.internal.tools
+Bundle-Description: IBM XML Web Services Tools, version ${bVersion}
```

Comment:
This is open liberty.  I believe IBM should be removed from the bundle description.